### DISCUSSION TITLE
[Schedule]Study schedule

### DISCUSSION THREAD

**Discussion by swiri021** [OpenKBC/multiple_sclerosis_proj#68] [Ideas]
@lacuss @kicheolkim Welcome Nicole to our project! I am just thinking about a schedule for next study round and study topic, and how about a cycle of analysis-engineering a month(One time for analysis and another time for engineering per month, total study meeting will be twice a month)? Additionally, I am wondering that how we run study time, like, paper review? code review? and we can do result review for the project. Let me know if you have a good idea!

**Comment by kicheolkim** [comment]
I didn't realize this discussions section. Welcome, Nicole!
I think we can do regular meet once a month now, and extra non-regular meetings when we need them. OR twice a month might be ok but different subject for each meeting?
Sounds good about paper review or code review. Probably we can start a result review using the current result.

**Reply by lacuss** [reply]
Sure, I agree on once a month and occasional meetings when needed. study or code review, absolutely. still figuring out AWS and codes, not getting hands-on codes really though. I would say just testing the water stage now. I think it would be great if we have a page for putting links for papers or articles to review.

**Reply by swiri021** [reply]
Good!, how about next week for code review? or other items ? I would like to suggest code review first.

**Comment by kicheolkim** [comment]
I'm probably not available next week. How about early November?

Kicheol

**Reply by swiri021** [reply]
I am good, maybe around first week of November ?

**Reply by lacuss** [reply]
sure- I am available anytime, just let me know

**Reply by kicheolkim** [reply]
November first week sounds good.

**Reply by swiri021** [reply]
Ok then, I am good on 11/4, 11/5, 11/6(Sat). 9:00PM 11/4 is the best for me. Any suggestion?

**Reply by lacuss** [reply]
11/4, 11/6 are both fine for me!

**Reply by swiri021** [reply]
@kicheolkim  @lacuss Let's fix the code review schedule on **11/4, 9:00PM**, and it will include the introduction of a recent update in the project. Kicheol, let us know if you are good.

**Reply by kicheolkim** [reply]
11/4 works for me. I think we all have a different time zone. Is 9 pm PST, EST, or CST?

**Reply by lacuss** [reply]
I thought it is the Chicago time zone where Jun lives? Anyways, I am okay with anytime later than 9 pm EST.

**Reply by swiri021** [reply]
Well then, I am living in CST, so how about 8PM CST ( 10PM EST, 6PM PST) ? Let me know!

**Reply by lacuss** [reply]
sure. I am in.

**Comment by kicheolkim** [comment]
It works for me.

Kicheol
